vignettes/articles/modify-functions.Rmd

```diff
@@ -0,0 +1,197 @@
+---
+title: "Modifier Functions"
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+```{r setup}
+library(gtsummary)
+theme_gtsummary_compact()
+```
+
+## Introduction
+
+Oftentimes when a table is created using {gtsummary} the default formatting does not exactly match a user's preferred formatting. To make modifications to tables after the table has been built, the `modify_*()` functions were created.
+
+In the following sections we briefly outline each of these functions that can be used to modify gtsummary tables.
+
+Many of the modifier functions share the common `columns` argument. When this argument is required, the column names provided should match their names in the table's `table_body` element (i.e. `x$table_body`). These column names can be printed to the console by running `show_header_names(x)`.
```

Comment:
Somewhere in this document, let's provide an example of `show_header_names()` with the default behaviour and showing the hidden columns. The hidden columns are often super useful when specifying `rows` arguments in these modify functions.

Reply:
Added